packages/TBC721/package.json

```diff
@@ -20,30 +20,30 @@
   ],
   "dependencies": {
     "@bitcoin-computer/lib": "^0.17.2-beta.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "ts-node": "^10.8.1",
-    "tslib": "~2.4.0"
+    "eslint-plugin-prettier": "^5.1.3",
+    "ts-node": "^10.9.2",
+    "tslib": "~2.6.2"
   },
   "devDependencies": {
-    "@types/bitcoin-computer__lib": "^0.17.1",
-    "@types/chai": "^4.3.1",
+    "@types/bitcoin-computer__lib": "^0.17.2",
```

Comment:
Same here, we should remove this devDependency to see if this change effectively work